pom.xml

```diff
@@ -219,6 +219,28 @@
                 <version>${calculationModuleVersion}</version>
                 <scope>test</scope>
             </dependency>
+
```

Comment:
are these needed? jaxb are all XML schema and binding dependencies.

Reply:
Agreed. Will remove this.